projects/plugins/boost/app/assets/src/js/features/page-cache/health/health.tsx

```diff
@@ -0,0 +1,25 @@
+import { Notice } from '@automattic/jetpack-components';
+import { useSingleModuleState } from '$features/module/lib/stores';
+import { usePageCacheDiagnosticDS } from '$lib/stores/page-cache';
+import { getDiagnosticData } from './lib/diagnostic';
+
+const Health = () => {
+	const [ pageCache ] = useSingleModuleState( 'page_cache' );
+	const data = usePageCacheDiagnosticDS();
+
+	// If the module is disabled, can it be enabled?
+	if ( pageCache?.active !== true && data?.canBeEnabled.status === false ) {
+		const diagnosticMessage = getDiagnosticData( data?.canBeEnabled.error );
+		if ( diagnosticMessage ) {
+			return (
+				<Notice level="warning" hideCloseButton={ true } title={ diagnosticMessage.title }>
+					{ diagnosticMessage.message }
+				</Notice>
+			);
+		}
+	}
+
+	// Don't mind me.
```

Comment:
LOL

I thought React got upset if you didn't return `null` here. I guess `undefined` is as good as a `null` :)

Reply:
Oh you're right! It either needs to return something or `null`. I've updated the code.